doc/source/parameters/cfd/physical_properties.rst

```diff
@@ -226,6 +226,44 @@ Conjugate heat transfer enables the addition of solid regions in which the fluid
     end
   end
 
+.. _immersed solid models:
+
+Immersed Solid Models
+~~~~~~~~~~~~~~~~~~~~~~
+
+Solid models can be used to affect specific behavior to immersed solids when ``lethe-fluid-sharp`` is used. At the moment, such a model is only available for the ``tracer`` multiphysics, but additional multiphysics will be included in the future.
+
+The immersed solid properties models are based on the signed distance function of the immersed solids, and therefore depend on the depth inside the solid. One of the intents of these models is to smoothen the transition of physical properties between the fluid and solid phase.
+
+The ``tracer diffusivity model`` parameter sets which diffusivity model is used. The default ``tracer diffusivity model`` is ``constant``, which uses a constant ``tracer diffusivity``. The alternative is ``immersed solid tanh``, whose parameters are defined as such:
+
+.. code-block:: text
+
+    subsection physical properties
+      set number of fluids = 1
+      subsection fluid 0
+        set tracer diffusivity model = constant
+        set tracer diffusivity       = 1
+      end
+      set number of solids = 1
+      subsection solid 0
+        set tracer diffusivity model = immersed solid tanh
+        subsection immersed solid
+          subsection tanh levelset
+            set tracer diffusivity inside    = 1
+            set tracer diffusivity outside   = 1
+            set tracer diffusivity thickness = 1
+          end
+        end
+      end
```

Comment:
I'm not sure I understand. Why do you have a subsectoin solid and then inside the subsection solid have a subsection immersed solid? Shouldnt the immersed solid be its own subsection?

Furthermore, here you specify the diffusivity inside and outside. But outside of the solid is the fluid. Consequently, are you not duplicated a parameter which is the fluid diffusivity. I think that's not a very wise decision. Essentially, you are duplicating the fluid diffusivity model twice by having the solid also specify it....

Reply:
The way I coded it yes, so we could have a fluid diffusivity, and a solid diffusivity that varies depending on the depth

Reply:
I changed it to only a fluid model, that can be based also on levelset (for IB methods). I restructured the parameters as well in this way